src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts

```diff
@@ -50,7 +52,9 @@ export class DotEditContentComponent implements OnInit, OnDestroy {
 
     contentletActionsUrl: SafeResourceUrl;
     pageState$: Observable<DotRenderedPageState>;
+    isEnterpriseLicense$: Observable<boolean>;
```

Comment:
This is a HUGE component, that's why in my last refeactor I remove this into the toolbar component, we should not bring this back here, we might want to use your new component inside the old toolbar component you're removing here, I don't want to make this component bigger in the opossite we're trying to make it smaller and easier to test.